src/cmd/commit.ts

```diff
@@ -79,6 +99,16 @@ export const commit = async (): Promise<void> => {
   } finally {
     cd(rootDir)
   }
+  if (!env.CI || isChart) {
+    const status = await getOtomiDeploymentStatus()
+    if (status !== 'deployed') {
+      await nothrow(
```

Comment:
Can we create functions for this stuff? Like `setDeploymentStatus`? And rename `getOtomiDeploymentStatus` to `getDeploymentStatus`? (As the word Otomi does not need to be in there as we are already in Otomi code base, so imo that is redundant.)

Reply:
yes